src/compas/geometry/primitives/shapes/cylinder.py

```diff
@@ -165,6 +169,37 @@ def volume(self):
         """Float: The volume of the cylinder."""
         return self.circle.area * self.height
 
+    def to_vertices_and_faces(self, **kwargs):
+        if 'u' in kwargs:
```

Comment:
same here: `u = kwargs.get('u', 10)`

Reply:
I see the problem. actually, for the function to "make sense", `u` should even be at least 3. The question now is, what default in what case?
`u` not in `kwargs` -> 10
`u in [None, <0, 0, 1, 2]` -> 3
?

Reply:
i think @gonzalocasas was suggesting the `.get` function is shorthand for what you wrote in a more verbose way. but now it is a different problem. perhaps just
```python
u = kwargs.get('u') or 0
if u < 3:
    raise ValueError('The value for u should be u > 3.')
```

Reply:
that sounds good, i would still do `u = kwargs.get('u') or 10` as a default (corrected typo)